armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -597,13 +622,12 @@ def updateDpaRate(self, blockList=None):
 
         if not hasDPA:
             return
-        self.r.core.p.peakGridDpaAt60Years = (
-            self.r.core.getMaxBlockParam(
-                "detailedDpaPeakRate", typeSpec=Flags.GRID_PLATE, absolute=False
-            )
-            * 60.0
-            * units.SECONDS_PER_YEAR
+
+        peakRate = self.r.core.getMaxBlockParam(
+            "detailedDpaPeakRate", typeSpec=Flags.GRID_PLATE, absolute=False
         )
+        self.r.core.p.peakGridDpaAt60Years = peakRate * 60.0 * units.SECONDS_PER_YEAR
```

Comment:
It is so odd IMO that this is a parameter in the framework... I am an advocate of moving it into our internal global flux plugin.

Reply:
Yeah, I noticed that too.

I would be in favor of moving this parameter in another PR very soon.  It _feels_ very case specific to me. Though we could get Nick and Jake's opinion on that.

Reply:
@jakehader @ntouran Any thoughts here?

Reply:
Personal opinion: It seems extremely specific and definitely odd to take a rate and multiply it by 60 years in _every_ case. Not every reactor even has a `Flags.GRID_PLATE`. You could store the rate as a parameter, and then somebody who wants to know the accumulated DPA can just... multiply by 60 years.

Devil's advocate: 60 years is a somewhat standard reactor life (40-year initial operating license + 20 year renewal), so you could argue that it's widely relevant.